frontend/TestRun/EventsTab.svelte

```diff
@@ -117,11 +117,29 @@
             return parsed;
         });
 
+        let sortedNemesises = nemesisData.sort((a, b) => a.start_time - b.start_time);
+        let nemesisIndex = 0;
+        for (let event of parsedEvents.sort((a, b) => a.time - b.time)) {
+            event.nemesis = undefined;
+                while(sortedNemesises[nemesisIndex]){
```

Comment:
You should correct the indentation of the while block and also fix spaces before `{` and after `}` characters (there should be a space in every case, e.g. `while (true) {`)